duckduckgo/DaxMailer
# Friends newsletter support

Support for unscheduled newsletters with custom content.

TBD:

Testing:

- Run `DAXMAILER_BASIC_AUTH_USER=ddg DAXMAILER_BASIC_AUTH_PASS=ddg bin/dev_server.sh -p 5555`
- Visit /s/friends
- You should be prompted for user / pass - only ddg:ddg will work
- You should be able to send yourself a test mail from this form
- Add subscribers from: /s/form?c=friends (Only your own address!!)
- Queue a newsletter from /s/friends
- Newsletter is in ~/ddgc/newsletter.txt
- `carton exec bin/subscriber_mail.pl --newsletter` should email queued newsletter to subscribers
- Newsletter file should be deleted after this.
